src/main/webapp/app/exam/exam-scores/exam-scores-average-scores-graph.component.html

```diff
@@ -1,5 +1,24 @@
 <div class="row">
-    <div class="col-md-7">
-        <canvas baseChart [height]="height" [datasets]="chartData" [labels]="barChartLabels" [options]="barChartOptions" [legend]="chartLegend" [chartType]="barChartType"></canvas>
+    <div #containerRef class="col-md-7" style="align-items: center">
+        <h6>{{ 'artemisApp.examScores.exerciseGroupTitle' | artemisTranslate: { groupTitle: this.averageScores.title } }}</h6>
+        <ngx-charts-bar-horizontal
+            [view]="[containerRef.offsetWidth, 100]"
+            [results]="ngxData"
+            [scheme]="ngxColor"
+            [xAxis]="true"
+            [yAxis]="true"
+            [xScaleMax]="xScaleMax"
+            [xAxisTickFormatting]="formatTicks"
+            (select)="onSelect($event)"
+        >
+            <ng-template #tooltipTemplate let-model="model">
+                <h6>{{ model.name }}</h6>
+                <span
```

Comment:
could you try to manually adjust the code here (the places that the commit automatically broke up these tags is quite ugly and sometimes it helps to manually readjust them and recommit^^

Reply:
I tried to adjust it in ddade2e but it does not seem to me that it had any effect😅